ARCHITECTURE.md

```diff
@@ -173,11 +168,12 @@ CodeRenderer
 ------------
 
 CodeRenderer is a standalone class that takes the result of the CodeGenerator class and constructs files.
-There are 3 files constructed:
+There are 4 files constructed:
```

Comment:
Up to ? as if you don't opt into the optional cmake file?

Reply:
Thanks, added "up to"